tests/conftest.py

```diff
@@ -25,7 +25,7 @@ def df_numeric_columns():
         1: ["London", "Manchester", "Liverpool", "Bristol"],
         2: [20, 21, 19, 18],
         3: [0.9, 0.8, 0.7, 0.6],
-        4: pd.date_range("2020-02-24", periods=4, freq="T"),
+        4: pd.date_range("2020-02-24", periods=4, freq="min"),
```

Comment:
could you remind me why we need this change?

Reply:
freq="T" is deprecated since version 2.2.0 scroll down to the red box -> [details](https://pandas.pydata.org/docs/user_guide/timeseries.html#timeseries-offset-aliases)